backend/app/api/ws/local_executor_namespace.py

```
@@ -0,0 +1,631 @@
+            {
+                "task_id": task_id,
+                "subtask_id": subtask_id,
+                "progress": progress,
+                "status": status,
+                "message": message,
+            },
+        )
+
+        return {"success": True}
+
+    async def on_task_result(self, sid: str, data: dict) -> dict:
```

Ruff (0.14.14)

Unused method argument: `sid`

(ARG002)